.drone.star

```diff
@@ -0,0 +1,50 @@
+  linux_cxx("TOOLSET=gcc COMPILER=g++-8 CXXSTD64=03,11,14, Job 8", "g++-8", packages="g++-8 g++-8-multilib linux-libc-dev:i386", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu1604:1", environment={'TOOLSET': 'gcc', 'COMPILER': 'g++-8', 'CXXSTD64': '03,11,14,17,03-gnu,11-gnu,14-gnu,17-gnu', 'CXXSTD32': '03,11', 'INSTRUCTION_SET': 'core2', 'DRONE_JOB_UUID': 'fe5dbbcea5'}, globalenv=globalenv),
+  linux_cxx("TOOLSET=gcc COMPILER=g++-9 CXXSTD64=03,11,14, Job 9", "g++-9", packages="g++-9 g++-9-multilib linux-libc-dev:i386", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu1804:1", environment={'TOOLSET': 'gcc', 'COMPILER': 'g++-9', 'CXXSTD64': '03,11,14,17,03-gnu,11-gnu,14-gnu,17-gnu', 'CXXSTD32': '03,11', 'INSTRUCTION_SET': 'core2', 'DRONE_JOB_UUID': '0ade7c2cf9'}, globalenv=globalenv),
+  linux_cxx("TOOLSET=gcc COMPILER=g++-10 CXXSTD64=03,11,14 Job 10", "g++-10", packages="g++-10 g++-10-multilib linux-libc-dev:i386", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu1804:1", environment={'TOOLSET': 'gcc', 'COMPILER': 'g++-10', 'CXXSTD64': '03,11,14,17,20,03-gnu,11-gnu,14-gnu,17-gnu,20-gnu', 'CXXSTD32': '03,11', 'INSTRUCTION_SET': 'core2', 'DRONE_JOB_UUID': 'b1d5781111'}, globalenv=globalenv),
+  linux_cxx("UBSAN=1 TOOLSET=gcc COMPILER=g++-10 CXXSTD64= Job 11", "g++-10", packages="g++-10", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu1804:1", environment={'UBSAN': '1', 'TOOLSET': 'gcc', 'COMPILER': 'g++-10', 'CXXSTD64': '03,11,14,17,20', 'INSTRUCTION_SET': 'core2', 'UBSAN_OPTIONS': 'print_stacktrace=1', 'BUILD_VARIANT': 'debug', 'LINKFLAGS': '-fuse-ld=gold', 'DRONE_JOB_UUID': '17ba079149'}, globalenv=globalenv),
+  linux_cxx("TOOLSET=gcc COMPILER=g++ CXXSTD64=03,11,14,17 Job 12", "g++", packages="", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu2004:1", environment={'TOOLSET': 'gcc', 'COMPILER': 'g++', 'CXXSTD64': '03,11,14,17,03-gnu,11-gnu,14-gnu,17-gnu', 'DRONE_JOB_UUID': '7b52009b64'}, globalenv=globalenv),
+  linux_cxx("TOOLSET=gcc COMPILER=g++ CXXSTD64=03,11,14,17 Job 13", "g++", packages="", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu2004:1", environment={'TOOLSET': 'gcc', 'COMPILER': 'g++', 'CXXSTD64': '03,11,14,17,03-gnu,11-gnu,14-gnu,17-gnu', 'DRONE_JOB_UUID': 'bd307a3ec3'}, globalenv=globalenv),
+  linux_cxx("TOOLSET=gcc COMPILER=g++ CXXSTD64=03,11,14,17 Job 14", "g++", packages="", buildtype="boost", buildscript="drone", image="cppalliance/droneubuntu2004:1", environment={'TOOLSET': 'gcc', 'COMPILER': 'g++', 'CXXSTD64': '03,11,14,17,03-gnu,11-gnu,14-gnu,17-gnu', 'DRONE_JOB_UUID': 'fa35e19212'}, globalenv=globalenv),
```

Comment:
These three are supposed to run on non-x86 CPUs, and I don't see this reflected in the arguments.